### PR TITLE
Faster socket connection timeouts in RequestProgram

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

I haven't tested this, but based on the `Socket` documentation this should improve user experience when running into issues with the connection with the remote PC as in #8.

The approach in #1 may be better, but at least this will help in the meantime.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Related question: is there a way to setup CI for URCaps?

File: src/main/java/com/fzi/externalcontrol/impl/RequestProgram.java [hunk 1]
```diff
@@ -29,6 +29,7 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.net.InetSocketAddress
```
Comment (fmauch) [inline src/main/java/com/fzi/externalcontrol/impl/RequestProgram.java:32]:
```suggestion
import java.net.InetSocketAddress;
```

**Comment by fmauch** [comment]
> Related question: is there a way to setup CI for URCaps?

Is #10 an answer to that question? :-)

**Comment by gavanderhoorn** [comment]
> > Related question: is there a way to setup CI for URCaps?
> 
> Is #10 an answer to that question? :-)

Awesome.

Can the artefacts from those builds actually be installed as well?

That would be really efficient for testing out PRs.

**Comment by fmauch** [comment]
> Can the artefacts from those builds actually be installed as well?

Yes, they should be usable.

**Pull request merged**